vectara/.env-template

```diff
@@ -0,0 +1,3 @@
+VECTARA_CUSTOMER_ID=12345567
+VECTARA_CORPUS_ID=1
+VECTARA_API_KEY=zwt_xxx
```

Comment:
Can you add `VECTARA_CONNECTOR_API_KEY`, this will secure the connector with a user-managed key to the `/search` endpoint